### PR TITLE
Feature/vrpp

### LINKED ISSUE #43: Add Virtual Reality Piloting Pod
The Virtual Reality Piloting Pod (VRPP) is not currently implemented, as far as I can tell, the construction rules are

> Construction Rules: The VRPP is an alternative form of the torsomounted cockpit from Tactical Operations (see pp. 300-301, TO), but weighs only 3 tons (rather than the 4 tons of a normal torso-mounted cockpit). Also unlike the standard torso-mounted cockpit, a VRPPmodified cockpit places 2 Sensor critical slots in the ’Mech’s head, and places 1 Life Support critical slot each in each of its 3 torso hit locations. The cockpit critical slot is then placed in the center torso.

and found in Interstellar Operations.

Availability of  (E/XXFX), prototype production year of 3052, only available to Battlemechs

### PR DESCRIPTION
**Pull request by WEKarnesky**
Adds the Virtual Reality Piloting Pod, As Described on Page 69 of Interstellar Operations.

Fixes #43

### CODE REVIEW

File: sswlib/src/main/java/states/stCockpitVirtualRealityPilotingPod.java [hunk 1]
```diff
@@ -0,0 +1,208 @@
+/*
+Copyright (c) 2008~2009, Justin R. Bengtson (poopshotgun@yahoo.com)
```
Comment (mrpopogod) [inline sswlib/src/main/java/states/stCockpitVirtualRealityPilotingPod.java]: Copyright probably should be updated.

**Comment by mrpopogod** [comment]
Code looks correct based on existing functionality, and confirmed the construction rules with the rulebooks.

**Comment by WEKarnesky** [comment]
@mrpopogod As requested:

[SSW_0.7.1.20191009--VRPP.zip](https://github.com/Solaris-Skunk-Werks/solarisskunkwerks/files/3715583/SSW_0.7.1.20191009--VRPP.zip)

**Comment by mrpopogod** [comment]
Testing out the build, criticals are added and removed appropriately as I select and deselect, and I get an informative error message if I already have a full head ejection system when I select it.  Availability in the program shows up as Experimental Tech for IS in Clan Invasion only, which matches the availability code in IO.

**Pull request merged**